From: Philip Hands <phil@hands.com>
Date: Tue, 16 Feb 2021 15:38:38 +0100
Subject: Debian tweaks for apache config

Note that we also sed vhosts.d to sites-available.d in
debian/rules
---
 etc/apache2/vhosts.d/openqa-ssl.conf.template | 14 +++++---------
 etc/apache2/vhosts.d/openqa.conf.template     |  1 +
 2 files changed, 6 insertions(+), 9 deletions(-)

diff --git a/etc/apache2/vhosts.d/openqa-ssl.conf.template b/etc/apache2/vhosts.d/openqa-ssl.conf.template
index d23a80a..ca48d92 100644
--- a/etc/apache2/vhosts.d/openqa-ssl.conf.template
+++ b/etc/apache2/vhosts.d/openqa-ssl.conf.template
@@ -1,12 +1,12 @@
 # sample apache config file for the ssl enabled openqa vhost.
 # At the very minimum you need to set the ServerName.
 
-# Note that this file will be ignored unless "SSL" is added to
-# APACHE_SERVER_FLAGS in /etc/sysconfig/apache2 (at least under
-# openSUSE).
+# There is a matching file nearby for non-ssl vhosts (openqa.conf.template) that
+# describes how to set things up, which also applies when setting up SSL.
+#
+# For SSL, you'll also need to load:
 
-<IfDefine SSL>
-<IfDefine !NOSSL>
+# a2enmod ssl
 
 <VirtualHost *:443>
 
@@ -26,7 +26,3 @@
     Include /etc/apache2/vhosts.d/openqa-common.inc
 
 </VirtualHost>
-
-</IfDefine>
-</IfDefine>
-
diff --git a/etc/apache2/vhosts.d/openqa.conf.template b/etc/apache2/vhosts.d/openqa.conf.template
index 98af74a..ae3b1e7 100644
--- a/etc/apache2/vhosts.d/openqa.conf.template
+++ b/etc/apache2/vhosts.d/openqa.conf.template
@@ -17,6 +17,7 @@
 # a2enmod proxy_http
 # a2enmod proxy_wstunnel
 # a2enmod rewrite
+# a2enmod expires
 
 <VirtualHost *:80>
 
